scripts/tool/generate_tool_package_template.py

```diff
@@ -33,7 +33,7 @@ def create_tool_project_structure(destination: str, package_name: str, tool_name
         class_name = convert_tool_name_to_class_name(tool_name)
 
     # Load templates
-    templates_abs_path = Path("scripts/tool/templates").resolve()
+    templates_abs_path = os.path.join(os.path.dirname(__file__), "templates")
```

Comment:
Please fix the flake8 issue

![image](https://github.com/microsoft/promptflow/assets/46446115/3e5a94ec-964c-4ae4-87ab-8220c99ad5d6)